src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -256,10 +264,11 @@ def HandleCommissioningComplete(nodeid, err):
             self._ChipStack.commissioningCompleteEvent.set()
             self._ChipStack.completeEvent.set()
 
-        def HandleOpenWindowComplete(nodeid: int, setupPinCode: int, setupCode: str, err: PyChipError) -> None:
+        def HandleOpenWindowComplete(nodeid: int, setupPinCode: int, manualCode: str, setupCode: str, err: PyChipError) -> None:
```

Comment:
Sugegst rename setupCode to setupQrCode

Reply:
Thank you for your suggestion, it has been revised